### PR TITLE
Enable backend statistics and match test properties with main

### PR DESCRIPTION
**Pull request by burmanm**

### CODE REVIEW

File: engine/src/main/resources/ispn-alerting-local.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     </local-cache>
 
     <!-- Backend for definitions, actions history, events and alerts -->
-    <local-cache name="backend">
+    <local-cache name="backend" statistics="true">
```
Comment (pilhuhn) [inline engine/src/main/resources/ispn-alerting-local.xml:29]: Would we want to enable statistics for the cache manager as well?
See https://infinispan.org/docs/dev/titles/configuring/configuring.html#statistics

**Review by pilhuhn — APPROVED** [review]

File: engine/src/main/resources/ispn-alerting-local.xml (hunk 1 above)
Comment (burmanm) [reply engine/src/main/resources/ispn-alerting-local.xml:29]: You mean at the cache-container level?

Comment (burmanm) [reply engine/src/main/resources/ispn-alerting-local.xml:29]: It adds some stuff, like: 

```
# HELP vendor_cache_manager_hawkular_alerts_cache_container_stats_average_write_time Cache container average number of milliseconds for all write operation in this cache container
# TYPE vendor_cache_manager_hawkular_alerts_cache_container_stats_average_write_time gauge
vendor_cache_manager_hawkular_alerts_cache_container_stats_average_write_time{node="nina-24401"} 0.0
# HELP vendor_cache_manager_hawkular_alerts_cache_container_stats_average_write_time_nanos Cache container average number of nanoseconds for all write operation in this cache container
# TYPE vendor_cache_manager_hawkular_alerts_cache_container_stats_average_write_time_nanos gauge
vendor_cache_manager_hawkular_alerts_cache_container_stats_average_write_time_nanos{node="nina-24401"} 526797.0
```

**Comment by burmanm** [comment]
The test failure is acceptable, it's because of disabling the data-trigger setting.

**Pull request merged**